server/sonar-web/src/main/js/apps/coding-rules/rule/rule-meta-view.js

```diff
@@ -107,7 +107,7 @@ export default Marionette.ItemView.extend(RuleFilterMixin).extend({
     return _.extend(Marionette.ItemView.prototype.serializeData.apply(this, arguments), {
       canWrite: this.options.app.canWrite,
       allTags: _.union(this.model.get('sysTags'), this.model.get('tags')),
-      permalink: '/coding_rules#rule_key=' + encodeURIComponent(this.model.id)
+      permalink: window.baseUrl + '/coding_rules#rule_key=' + encodeURIComponent(this.model.id)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)